bin/barnard59-docs.js

```diff
@@ -8,19 +8,27 @@ const program = new Command()
 program
   .option('--operation-template <filename>', 'ES6 template for an operation')
   .option('--package-template <filename>', 'ES6 template for a package')
-
-program.parse(process.argv)
-
-const output = []
-
-for (const name of program.args) {
-  const info = await packageInfo(name)
-  const manifest = await packageManifest(name)
-
-  output.push(infoToMarkdown(info, program.opts()))
-  output.push(manifestToMarkdown(manifest, program.opts()))
-}
-
-const markdown = output.join('\n')
-
-process.stdout.write(markdown)
+  .option('--local', 'Treat arguments as paths to package directories')
+  .option('--manifest <manifest>', 'Path to manifest.ttl relative to package.json', 'manifest.ttl')
```

Comment:
The `manifest.ttl` file is always expected in the root of the package. That should be the case for npm dependencies, but also for definitions in the same project. I think this argument is not required. Do you have any use cases in mind where this could be used?

Reply:
probably not. happy to remove